HerPublicWebsite.ManagementShell/FakeReferralGenerator.cs

```diff
@@ -0,0 +1,148 @@
+﻿using Bogus;
+using HerPublicWebsite.BusinessLogic.Models;
+using HerPublicWebsite.BusinessLogic.Models.Enums;
+
+namespace HerPublicWebsite.ManagementShell;
+
+public interface IFakeReferralGenerator
+{
+    IEnumerable<ReferralRequest> GenerateFakeReferralRequests(int count, int startId);
+}
+
+public class FakeReferralGenerator : IFakeReferralGenerator
+{
+    public IEnumerable<ReferralRequest> GenerateFakeReferralRequests(int count, int startId)
+    {
+        var id = startId;
+        var custodianCodes = LocalAuthorityData.LocalAuthorityDetailsByCustodianCode.Keys;
+
+        var referralsSchema = new Faker<ReferralRequest>("en_GB")
+            .RuleFor(rr => rr.Id, _ => id++)
```

Comment:
Does this not auto-increment?

Reply:
it does, however the code runs into issues generating the HUG2xxxxxx ID since this assumed an ID has already been generated. to make it work I'd need to insert all into database, auto generate ID, then re-query them & generate the HUG2 ID

Reply:
Would that be difficult? Appreciate it's an extra update, but that's not a huge worry for me, and messing around with the auto-incrementing ID can be a recipe for trouble in my experience.

Reply:
have made this change now